ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md

```diff
@@ -1156,6 +1184,77 @@
                 #   - 1-10
               - range: <str; required; unique>
                 mac_address_maximum: <int>
+          trunk:
+
+            # VLAN ID or range of VLAN IDs.
+            # Warning: This should not be combined with `ethernet_interface.mode = trunk`.
+            allowed_vlans: <str>
+
+            # Set native VLAN when interface is in trunking mode.
+            # Warning: This should not be combined with `ethernet_interfacse[].native_vlan`.
+            native_vlan: <int; 1-4094>
+
+            # If setting both native_vlan and native_vlan_tag, native_vlan_tag takes precedence.
+            # Warning: This should not be combined with `ethernet_interfacse[].native_vlan_tag`.
+            native_vlan_tag: <bool>
+
+            # Enable secondary VLAN mapping for a private vlan.
+            # Warning: This should not be combined with `ethernet_ineterfaces[].trunk_private_vlan_secondary`.
+            private_vlan_secondary: <bool>
+            groups:
+
+                # Trunk group name.
+                # Warning: This should not be combined with `ethernet_ineterfaces[].trunk_groups`.
+              - <str>
+
+          # Set VLAN when interface is in access mode.
+          # Warning: This should not be combined withe `ethernet_interfaces[].mode = access/dot1q-tunnel`.
+          access_vlan: <int>
```

Comment:
```
wan1-dc1(config-if-Et1)#switchport access vlan ?
  dynamic   Dynamic (non-persistent) access VLAN setting
  <1-4094>  Identifier for a Virtual LAN

wan1-dc1(config-if-Et1)#switchport access vlan dynamic ?
  <1-4094>  Identifier for a Virtual LAN

wan1-dc1(config-if-Et1)#switchport access vlan dynamic
```
add the limit plus if possible to add dynamic

Reply:
I tried `switchport access vlan dynamic 10` but it does not renders it in config

Reply:
Correct. Dynamic vlans are not in config. It is a way to set a port in a vlan without saving that information in the config. We do not need to model the dynamic option now.